### PR TITLE
fix: Update with generated code from protobufjs version 7.0.0

### PR DESCRIPTION
**Pull request by tom-andersen**
This is the result of running update.sh script .... I am very confused as to purpose of this script, since it appears to overlap googleapis-gen.

### CODE REVIEW

**Review by dconeybe — APPROVED** [review]

**Review by dconeybe — CHANGES_REQUESTED** [review]
I'm withdrawing my approval because we don't want stuff from the googleapis preview branch going into the main branch of this repo.

**Comment by alexander-fenster** [comment]
To give you some context, the initial idea of `update.sh` was to split protos into smaller files, and load them separately as needed, which should improve cold start time. This is a unique thing, no other client libraries do this. I haven't looked at why the preview features get included though, the script might need some fixing. Feel free to ping me if you want extra 👀 to look at it.

File: dev/protos/update.sh [hunk 1]
```diff
@@ -38,7 +38,7 @@ PBTS="$(npm bin)/pbts"
 pushd "$WORK_DIR"
 
 # Clone necessary git repos.
-git clone --depth 1 https://github.com/googleapis/googleapis.git
+git clone -b preview --depth 1 https://github.com/googleapis/googleapis.git
```
Comment (alexander-fenster) [inline dev/protos/update.sh:41]: Why `-b preview`?

**Comment by alexander-fenster** [comment]
In #1765 I updated protos with the latest updates from googleapis. I'm assuming this PR could be closed now.

**Pull request closed without merging**